MonoGame.Framework/Utilities/FileHelpers.cs

```diff
@@ -3,12 +3,15 @@
 // file 'LICENSE.txt', which is part of this source code package.
 
 using System;
-using System.IO;
-
+using System.IO;
+using System.Text;
+
 namespace Microsoft.Xna.Framework.Utilities
 {
     internal static class FileHelpers
-    {
+    {
+        private const string UrlSafeChars = "._-;/?:@&=+&,";

```

Comment:
You are right, I included the query URL components which I shouldn't have, fixed now.